airtime_mvc/application/controllers/LibraryController.php

```diff
@@ -447,20 +442,9 @@ public function editFileMdAction()
             }
 
             if ($form->isValid($serialized)) {
-
-                $formValues = $this->_getParam('data', null);
-                $formdata = array();
-                foreach ($formValues as $val) {
-                    $formdata[$val["name"]] = $val["value"];
-                }
-                $file->setDbColMetadata($formdata);
-
-                $data = $file->getMetadata();
-
-                // set MDATA_KEY_FILEPATH
-                $data['MDATA_KEY_FILEPATH'] = $file->getFilePath();
-                Logging::info($data['MDATA_KEY_FILEPATH']);
-                Application_Model_RabbitMq::SendMessageToMediaMonitor("md_update", $data);
+                // Sanitize any incorrect metadata that slipped past validation
+                FileDataHelper::sanitizeData($serialized);
```

Comment:
I think we should sanitize before we validate. What if the form validator bounces something that we could have sanitized anyways?
(I can move this line just above the if statement)